engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -2562,13 +2572,19 @@ protected void initDmnEngine() {
 
   protected void initExpressionManager() {
     if (expressionManager == null) {
-      expressionManager = new ExpressionManager(beans);
+      expressionManager = new JuelExpressionManager(beans);
     }
 
-    // add function mapper for command context (eg currentUser(), currentUserGroups())
-    expressionManager.addFunctionMapper(new CommandContextFunctionMapper());
-    // add function mapper for date time (eg now(), dateTime())
-    expressionManager.addFunctionMapper(new DateTimeFunctionMapper());
+    // it's better to add FunctionMapper when create default JuelExpressionManager above,
+    // but we leave it here for backward compatibility
```

Comment:
You are right with this comment in general, but given the next line, we can remove these here, I believe. The next line already explains in a constructive way why we do this here and not within the if-clause above.
```suggestion
```

Reply:
sorry for misunderstanding, done